docs/getting-started.md

```diff
@@ -0,0 +1,188 @@
+
+## Assumed knowledge
+
+To get the most out of this getting started guide, make sure you have an understanding of:
+
+- The basics of Great Expectations. See [Try GX Core](https://docs.greatexpectations.io/docs/core/introduction/try_gx/).
+- Airflow fundamentals, such as writing DAGs and defining tasks. See [Get started with Apache Airflow](https://www.astronomer.io/docs/learn/get-started-with-airflow/).
+- Airflow Operators. See [Operators 101](https://www.astronomer.io/docs/learn/what-is-an-operator/).
+- Airflow connections. See [Managing your Connections in Apache Airflow](https://www.astronomer.io/docs/learn/connections/).
+
+## Install the provider and dependencies
+
+1. Install the provider.
+
+   ```bash
+   pip install airflow-provider-great-expectations 
+   ```
+2. (Optional) Install additional dependencies for the data sources you’ll use. For example, to install the optional Snowflake dependency, use the following command.
+
+   ```bash
+   pip install "airflow-provider-great-expectations[snowflake]"
+   ```
+   The following backends are supported as optional dependencies:
+      - `athena`
+      - `azure`
+      - `bigquery`
+      - `gcp`
+      - `mssql`
+      - `postgresql`
+      - `s3`
+      - `snowflake`
+      - `spark`
+
+## Configure an Operator
+
+After deciding [which Operator best fits your use case](#operator-use-cases), follow the Operator-specific instructions below to configure it.
+
+### Data Frame Operator
+
+1. Import the Operator.
+
+    ```python
+    from great_expectations_provider.operators.validate_dataframe import (
+        GXValidateDataFrameOperator,
+    )
+    ```
+
+2. Instantiate the Operator with required and optional parameters.
+
+    ```python
+    from typing import TYPE_CHECKING
+
+    if TYPE_CHECKING:
+        from pandas import DataFrame
+    
+    def my_data_frame_configuration(): DataFrame:
+        import pandas as pd  # airflow best practice is to not import heavy dependencies in the top level
+        return pd.read_csv(my_data_file)
+    
+    my_data_frame_operator = GXValidateDataFrameOperator(
+        task_id="my_data_frame_operator",
+        configure_dataframe=my_data_frame_configuration,
+        expect=my_expectation_suite,
+    )
+    ```
+
+    - **`task_id`**: alphanumeric name used in the Airflow UI and GX Cloud.
+    - **`configure_dataframe`**: function that returns a DataFrame to pass data to the Operator.
+    - **`expect`**: either a [single Expectation](https://docs.greatexpectations.io/docs/core/define_expectations/create_an_expectation) or an [Expectation Suite](https://docs.greatexpectations.io/docs/core/define_expectations/organize_expectation_suites) to validate against your data. 
+    - **`result_format` (optional)**: dictionary that sets the [verbosity of returned Validation Results](https://docs.greatexpectations.io/docs/core/trigger_actions_based_on_results/choose_a_result_format/). Defaults to `SUMMARY`.
```

Comment:
GX Core does technically accept a dictionary as this value, but let's prefer the simpler string here: `Literal["BOOLEAN_ONLY", "BASIC", "SUMMARY", "COMPLETE"] `